src/Components/Header.js

```diff
@@ -5,47 +5,83 @@ import {
   View,
   Text,
   TouchableOpacity,
+  ScrollView,
 } from 'react-native';
 import SistemaButton from '../Components/SistemaButton';
-import SistemaLogo from '../../assets/sistemaLogo.svg';
+import SistemaLogo from '../../assets/SistemaLogo.svg';
```

Comment:
change back to sistemaLogo.svg
i think maybe pulling from master doesn't cut it bc of git's weirdness, so you might have to just manually change it locally